src/module.ts

```diff
@@ -25,13 +28,26 @@ export default defineNuxtModule<ModuleOptions>({
     siteKey: nuxt.options.dev ? '1x00000000000000000000AA' : undefined,
     addValidateEndpoint: false,
   }),
-  async setup (options, nuxt) {
+  async setup(options, nuxt) {
     const siteKey = options.siteKey || nuxt.options.runtimeConfig.public?.turnstile?.siteKey
     if (!siteKey) {
       console.warn('`@nuxtjs/turnstile` is disabled as no site key was provided.')
       return
     }
 
+    const secretKeyPath =
+      options.secretKeyPath || nuxt.options.runtimeConfig.turnstile?.secretKeyPath
+    let secretKeyFromPath
+    if (secretKeyPath) {
+      secretKeyFromPath = fs.existsSync(secretKeyPath)
+        ? fs.readFileSync(secretKeyPath, 'utf-8')
+        : undefined
+    }
```

Comment:
```suggestion
    const secretKeyFromPath = secretKeyPath && fs.existsSync(secretKeyPath)
        ? fs.readFileSync(secretKeyPath, 'utf-8')
        : undefined
```
(just typed, might need to be changed for the linter)

It's almost always possible to use `const` only. For me a case where I want to use `let` always makes me think twice if I really need it. Mostly style preference only though.

Reply:
Applied that 👍

Reply:
This thread can be resolved :+1: